replication.py

```diff
@@ -53,26 +63,35 @@ def rule_replicate_batch(ctx, verbose):
     if is_replication_blocked_by_admin(ctx):
         log.write(ctx, "Batch replication job is stopped")
     else:
-        log.write(ctx, "Batch replication job started")
-
-        # Get list of data objects scheduled for replication.
-        iter = genquery.row_iterator(
-            "ORDER(DATA_ID), COLL_NAME, DATA_NAME, META_DATA_ATTR_VALUE, DATA_RESC_NAME",
-            "META_DATA_ATTR_NAME = '{}'".format(attr),
-            genquery.AS_LIST, ctx
-        )
+        log.write(ctx, "Batch replication job started - balance id: {}-{}".format(balance_id_min, balance_id_max))
+
+        # Get list up to 1000 data objects scheduled for replication.
+        iter = list(genquery.Query(ctx,
+                    ['ORDER(DATA_ID)', 'COLL_NAME', 'DATA_NAME', 'META_DATA_ATTR_VALUE', 'DATA_RESC_NAME'],
+                    "META_DATA_ATTR_NAME = '{}'".format(attr),
+                    offset=0, limit=1000, output=genquery.AS_LIST))
```

Comment:
We might want to adjust the batch sizes of the replication and revision job based on experiences with using the new implementation in production. Can we extract configuration parameters for the limit values of the replication and revision job?

Reply:
good thinking 👍
This is implemented